app/src/main/res/values/strings.xml

```diff
@@ -318,6 +318,9 @@ Please report any problems you encounter via Github.</string>
     <string name="run_conditions_title">Run Conditions</string>
     <string name="run_conditions_summary">Use the following options to decide when Syncthing will run.</string>
 
+    <string name="run_always_title">Run always</string>
+    <string name="run_always_summary">Ignore run conditions and run always. Use this if you want to manually control running of the app.</string>
```

Comment:
How does this give manual control? It means it runs all the time, no control whatsover, right? Seems like just the first sentence would cut it.

Reply:
The manual control is then exercised by starting and exiting the app. I agree that this option is badly worded all together. I guess the clearest labeling would be "Enable Run Conditions" as title with the default being true (to keep the behavior from before this PR unless explicitly changed). If set to false all run condition options are greyed out (like you mentioned in your other comment) and the run conditions are ignored What do you think? Thanks for the review!

Reply:
makes sense